app/jobs/ingest_pulfa_job.rb

```diff
@@ -40,6 +39,18 @@ def ingest
       r.save!
     end
 
+    def work_attributes
+      {
+        title: [@mets.xpath("//mets:structMap/mets:div/@LABEL").first.value],
```

Comment:
Yes, this is the background job for ingesting the based on the legacy METS files.